docs/source/conf.py

```diff
@@ -731,6 +731,8 @@
     "is_algebraic_expression",
     "is_bool_expr",
     "is_dim",
+    # torch.fx.passes.tensorify_python_scalars
+    "tensorify_python_scalars",
```

Comment:
Adding new entries here is no ok.
Please make sure that your function is properly documented or made private if you don't want it visible to the end user.

In this case, you are most likely missing the appropriate .rst entry to show this function on the docs website.